lib/elixir_analyzer/test_suite/captains_log.ex

```diff
@@ -32,6 +39,9 @@ defmodule ElixirAnalyzer.TestSuite.CaptainsLog do
     called_fn module: :rand, name: :uniform
     comment Constants.captains_log_use_rand_uniform()
     suppress_if Constants.solution_deprecated_random_module(), :fail
+    # TODO: support multiple suppress_if
+    # suppress_if Constants.captains_log_do_not_use_rand_uniform_real(), :fail
+    # suppress_if Constants.captains_log_do_not_use_enum_random(), :fail
```

Comment:
Currently this will return two comments about using `:rand.uniform`. To fix that, we would need: https://github.com/exercism/elixir-analyzer/issues/327

Reply:
I just opened  #329, maybe we can tackle it first, then fix the TODOs

Reply:
Done: [`d94561e` (#328)](https://github.com/exercism/elixir-analyzer/pull/328/commits/d94561ea09db17a09d1493358b203e60532c410e)